modules/database-commons/src/main/java/org/testcontainers/ext/ScriptUtils.java

```diff
@@ -210,6 +214,30 @@ public static boolean containsSqlScriptDelimiters(String script, String delim) {
 		return false;
 	}
 
+	/**
+	 * Load script from classpath and apply it to the given database
+	 *
+	 * @param databaseDelegate database delegate for script execution
+	 * @param initScriptPath   the resource to load the init script from
+	 */
+	public static void runInitScript(DatabaseDelegate databaseDelegate, String initScriptPath) {
```

Comment:
This method can also be used in ContainerDatabaseDriver.runInitScriptIfRequired(...) to replace almost all of the method's code, but I didn't do it to have diff smaller.